<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FOCUS-QA-133" name="com.focus.testcases.ModifiedScriptForDeleteTest" tests="18" failures="1" timestamp="3 Nov 2021 12:03:11 GMT" time="451.101" errors="0">
  <testcase name="checkSavingSalesInvoiceVATVoucher1WithQuantityAs11AfterAddingRow2WithBatchNoAsBatchPVAT2InPurchaseVoucherVAT1" time="33.781" classname="com.focus.testcases.ModifiedScriptForDeleteTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.ModifiedScriptForDeleteTest.checkSavingSalesInvoiceVATVoucher1WithQuantityAs11AfterAddingRow2WithBatchNoAsBatchPVAT2InPurchaseVoucherVAT1(ModifiedScriptForDeleteTest.java:802)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkSavingSalesInvoiceVATVoucher1WithQuantityAs11AfterAddingRow2WithBatchNoAsBatchPVAT2InPurchaseVoucherVAT1 -->
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByAddingRow2WithSameQuantityAndBatchNoForTwentySeventhScenario" time="35.884" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkSavingSalesInvoiceVATVoucher1WithQuantityAs11AfterAddingRow2WithBatchNoAsBatchPVAT1InPurchaseVoucherVAT1" time="34.774" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkReSavingSalesInvoiceVATVoucher1AfterDeletingARow2InPurchaseVoucherVAT1" time="19.917" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkSuspendingSalesInvoiceVATVoucher1ForTwentyEightScenario" time="15.106" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkSuspendingSalesInvoiceVATVoucher1ForTwentyFourthScenario" time="14.584" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByDeleting2ndAfterSuspendingSalesInvoice" time="24.527" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkSuspendingSalesInvoiceVATVoucher1ForTwentySixthScenario" time="14.956" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByDeleting2ndRowWhichHasConsumedInSalesInvoice1" time="23.078" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByDeletingRow2WhichHasConsumedFullyInSalesInvoiceAfterSuspendingSalesInvoice" time="24.065" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByDeletingRow2WhichHasConsumedFullyInSalesInvoice" time="20.640" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByDeleting2ndRowWhichHasConsumedInSalesInvoice" time="23.955" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByAddingRow2WithSameQuantityAndDifferentBatchNo" time="35.190" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByAdding2ndRowWithSameQuantityAndDifferentBatchNo" time="35.628" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkReSavingSalesInvoiceVATVoucher1AfterAddingRow2WithBatchNoAsBatchPVAT2InPurchaseVoucherVAT1" time="30.152" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkReSavingSalesInvoiceVATVoucher1AfterDeleting2ndRowInPurchaseVoucherVAT1" time="20.139" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkEditingAndSavingPurchaseVoucherVATVoucher1ByDeleting2ndWhichHasConsumedInSalesInvoiceAfterSuspendingSalesInvoice" time="24.338" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
  <testcase name="checkReSavingSalesInvoiceVATVoucherAfterDeleting2ndRowInPurchaseVoucherVAT1" time="20.387" classname="com.focus.testcases.ModifiedScriptForDeleteTest"/>
</testsuite> <!-- com.focus.testcases.ModifiedScriptForDeleteTest -->
